File: python/8-kyu/how-old-will-i-be-in-2099/how-old-will-i-be-in-2099.py
def calculate_age(year_of_birth, current_year):
    total = abs(current_year - year_of_birth)
    
    y = 'years'
    if total == 1:
        y = 'year'
        
    if year_of_birth < current_year:
        return f"You are {total} {y} old."
    elif year_of_birth > current_year:
        return f"You will be born in {total} {y}."
    else:
        return "You were born this very year!"

File: javascript/8-kyu/how-old-will-i-be-in-2099/how-old-will-i-be-in-2099.js
function  calculateAge(bYear, eYear) {  
  // enter your code here.
  let age = Math.abs(eYear - bYear);
  if (bYear < eYear && age === 1) {
    return `You are ${age} year old.`
  } else if (bYear > eYear && age === 1) {
    return `You will be born in ${age} year.`
  } else if (bYear < eYear) {
    return `You are ${age} years old.`
  } else if (bYear > eYear) {
    return `You will be born in ${age} years.`
  } else {
    return `You were born this very year!`
  }
}
​
​